osquery/devtools/shell.cpp

```diff
@@ -1083,7 +1088,7 @@ inline void meta_schema(int nArg, char** azArg) {
       fprintf(stdout,
               "CREATE TABLE %s%s;\n",
               table.c_str(),
-              osquery::columnDefinition(response, true).c_str());
+              osquery::columnDefinition(response, true, false).c_str());
```

Comment:
I'd like to suggest you make such calls more explicit in terms of passing arguments. Smth like this:
```c++
auto const aliases = true;
auto const is_extension = false;
osquery::columnDefinition(response, aliases, is_extension).c_str());
```
What do you think?